### PR TITLE
Adding Processing Module

### PR DESCRIPTION
**Pull request by gwaygenomics**
~**This pull request is a work in progress** (I still need to add documentation and analysis pipeline instructions)~

The following pull request adds, fixes, and/or modifies:

* Adding a new analysis module `2.process-data`. This module reads in data from `1.download-data` and processes it for input into single cell classifiers.
* [A] `0.random-test-split.ipynb` - Feeds in the EBImage data and splits off 10% randomly, stratified by MOA target class
* [A] `1.process-ebimage-features.ipynb` - Uses [pycytominer](https://github.com/cytomining/pycytominer/tree/3138f3bb0eb19a4cab5bd93e2faf47fce953d82f) to select EBImage features and normalize training and test sets
* [A] `process-data-pipeline.sh` - bash script specifying how to reproduce data processing
* [A] `README.md` - adding specifically for the processing data module
* [A] Processed Training and Test sets - these are gzipped and are not large (since they contain fewer features) so they are included here
* [M] `environment.yml` - downgrading python 3.8 to python 3.7.3 to enable sklearn 0.21.3 (for pycytominer). See scikit-learn/scikit-learn#15328.
* [M] `1.download-data/README.md` - minor documentation improvements

I performed the following prior to filing this pull request:

- [x] Tested that my change does not break the analysis pipeline
- [x] Ran a linter through my code
- [x] Update environment dependencies if my code introduces a new package

### CODE REVIEW

**Comment by gwaygenomics** [comment]
Thanks @wagenrace - PR ready for review
